internal/provider/data_source_api_token_permission_groups.go

```diff
@@ -32,6 +32,11 @@ func dataSourceCloudflareApiTokenPermissionGroupsRead(ctx context.Context, d *sc
 		return diag.FromErr(fmt.Errorf("error listing API Token Permission Groups: %w", err))
 	}
 
+	err = d.Set("raw_permissions", permissions)
```

Comment:
this field doesn't exist on the schema so it won't have any effect.

what i would recommend we do here is add two new fields to the schema, `account` and `zones` each with their respective permissions that sit under it. that way, if someone wants to target `Foo Permission` on their zone, it's pretty explicit as `data.cloudflare_api_token_permissions.zone["Foo Permission"]`